parentheses/algorithm.go

```diff
@@ -0,0 +1,27 @@
+// Package implements function BalancedString, that verifies if the given string is a balanced sequence of brackets
+package parentheses
+
+// BalancedString verifies if the given string is a balanced sequence of brackets
+var brackets = map[string]string{"{": "}", "(": ")", "[": "]"}
+
+func BalancedString(s string) bool {
+	stack := make([]string, 0, len([]rune(s)))
+
+	for i := 0; i < len(s); i++ {
+		v := string(s[i])
```

Comment:
you're counting the number of runes, but iterating over bytes

Reply:
you should iterate over runes as well

Reply:
```go
for i := 0; i < len(s); i++ {
    v := string(s[i])
```
+ it's much easier to just `range` over `string`

Reply:
Done!